SD-K9/src/app/models/vertical-transport.enum.model.ts

```diff
@@ -0,0 +1,5 @@
+export enum VerticalTransport {
+    ESCALATOR = 'BICYCLING',
+    ELEVATOR = 'DRIVING',
+    STAIRS = 'TRANSIT'
```

Comment:
```suggestion
    ESCALATOR = 'ESCALATOR ',
    ELEVATOR = 'ELEVATOR ',
    STAIRS = 'STAIRS '
```
Should it be like this?

Reply:
yes! my mistake